### ISSUE TITLE
New submissions for Mon,  4 Dec 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#160]
# New submissions for Mon,  4 Dec 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Evolution and final fate of solar metallicity stars in the mass range  7-15 Msun. I. The transition from AGB to SAGB stars, Electron Capture and  Core Collapse Supernovae progenitors
 - **Authors:** Marco Limongi, Lorenzo Roberti, Alessandro Chieffi, Ken'ichi Nomoto
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2312.00107
 - **Pdf link:** https://arxiv.org/pdf/2312.00107
 - **Abstract**
 According to a standard initial mass function, stars in the range 7-12 Msun constitute ~50% (by number) of the stars more massive than ~7 Msun, but, in spite of this, their evolutionary properties, and in particular their final fate, are still scarcely studied. In this paper we present a detailed study of the evolutionary properties of solar metallicity, non rotating stars in the range 7-15 Msun, from the pre main sequence phase up to the presupernova stage or up to an advanced stage of the thermally pulsing phase, depending on the initial mass. We find that (1) the 7.00 Msun develops a degenerate CO core and evolves as a classical AGB star in the sense that it does not ignite the C burning reactions; (2) stars with the initial mass M >= 9.22 Msun end their life as core collapse supernovae; (3) stars in the range 7.50 <= M/Msun <= 9.20 develop a degenerate ONeMg core and evolve through the thermally pulsing SAGB phase; 4) stars in the mass range 7.50 <= M/Msun <= 8.00 end their life as hybrid CO/ONeMg- or ONeMg- WD; (5) stars with the initial mass in the range 8.50 <= M/Msun <= 9.20 may potentially explode as electron capture supernovae.
#### $\texttt{tdescore}$: An Accurate Photometric Classifier for Tidal  Disruption Events
 - **Authors:** Robert Stein, Ashish Mahabal, Simeon Reusch, Matthew Graham, Mansi M. Kasliwal, Marek Kowalski, Suvi Gezari, Erica Hammerstein, Szymon J. Nakoneczny, Matt Nicholl, Jesper Sollerman, Sjoert van Velzen, Yuhan Yao, Russ R. Laher, Ben Rusholme
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2312.00139
 - **Pdf link:** https://arxiv.org/pdf/2312.00139
 - **Abstract**
 Optical surveys have become increasingly adept at identifying candidate Tidal Disruption Events (TDEs) in large numbers, but classifying these generally requires extensive spectroscopic resources. We here present $\texttt{tdescore}$, a simple photometric classifier that is trained using a systematic census of $\sim$3000 nuclear transients from the Zwicky Transient Facility (ZTF). The sample is highly imbalanced, with TDEs representing $<$2% of the total. $\texttt{tdescore}$ is nonetheless able to reject non-TDEs with 99.6% accuracy, yielding a sample of probable TDEs with completeness of 77.0% and a purity of 80.3%. $\texttt{tdescore}$ is thus substantially better than any available TDE photometric classifier scheme in the literature, and performs comparably well to the single-epoch spectroscopy as a method for classifying ZTF nuclear transients, despite relying solely on ZTF data and multi-wavelength catalogue crossmatching. In a novel extension, we use 'SHapley Additive exPlanations' (SHAP) to provide a human-readable justification for each individual $\texttt{tdescore}$ classification, enabling users to understand and form opinions about the underlying classifier reasoning. $\texttt{tdescore}$ serves as a model for photometric identification of TDEs with time-domain surveys, such as the upcoming Rubin observatory.
#### Atmospheric metallicity and C/O of HD 189733 b from high-resolution  spectroscopy
 - **Authors:** Luke Finnerty, Jerry W. Xuan, Yinzi Xin, Joshua Liberman, Tobias Schofield, Michael P. Fitzgerald, Shubh Agrawal, Ashley Baker, Randall Bartos, Geoffrey A. Blake, Benjamin Calvin, Sylvain Cetre, Jacques-Robert Delorme, Greg Doppman, Daniel Echeverri, Chih-Chun Hsu, Nemanja Jovanovic, Ronald A. López, Emily C. Martin, Dimitri Mawet, Evan Morris, Jacklyn Pezzato, Jean-Baptiste Ruffio, Ben Sappey, Andrew Skemer, Taylor Venenciano, J. Kent Wallace, Nicole L. Wallack, Jason J. Wang, Ji Wang
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2312.00141
 - **Pdf link:** https://arxiv.org/pdf/2312.00141
 - **Abstract**
 We present high-resolution $K$-band emission spectra of the quintessential hot Jupiter HD 189733 b from the Keck Planet Imager and Characterizer (KPIC). Using a Bayesian retrieval framework, we fit the dayside pressure-temperature profile, orbital kinematics, mass-mixing ratios of H$_2$O, CO, CH$_4$, NH$_3$, HCN, and H$_2$S, and the $\rm ^{13}CO/^{12}CO$ ratio. We measure mass fractions of $\rm \log H_2O = -2.0^{+0.4}_{-0.4}$ and $\rm \log CO = -2.2^{+0.5}_{-0.5}$, and place upper limits on the remaining species. Notably, we find $\rm \log CH_4 < -4.5$ at 99\% confidence, despite its anticipated presence at the equilibrium temperature of HD 189733 b assuming local thermal equilibrium. We make a tentative ($\sim3\sigma$) detection of $\rm ^{13}CO$, and the retrieved posteriors suggest a $\rm ^{12}C/^{13}C$ ratio similar to or substantially less than the local interstellar value. The possible $\rm ^{13}C$ enrichment would be consistent with accretion of fractionated material in ices or in the protoplanetary disk midplane. The retrieved abundances correspond to a substantially sub-stellar atmospheric $\rm C/O = 0.3\pm0.1$, while the carbon and oxygen abundances are stellar to slightly super-stellar, consistent with core-accretion models which predict an inverse correlation between C/O and metallicity. The specific combination of low C/O and high metallicity suggests significant accretion of solid material may have occurred late in the formation process of HD 189733 b.
#### The ZTF Source Classification Project: III. A Catalog of Variable  Sources
 - **Authors:** Brian F. Healy, Michael W. Coughlin, Ashish A. Mahabal, Theophile J. du Laz, Andrew Drake, Matthew J. Graham, Lynne A. Hillenbrand, Jan van Roestel, Paula Szkody, LeighAnna Zielske, Mohammed Guiga, Muhammad Yusuf Hassan, Jill L. Hughes, Guy Nir, Saagar Parikh, Sungmin Park, Palak Purohit, Umaa Rebbapragada, Draco Reed, Avery Wold, Joshua S. Bloom, Frank J. Masci, Reed Riddle, Roger Smith
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2312.00143
 - **Pdf link:** https://arxiv.org/pdf/2312.00143
 - **Abstract**
 The classification of variable objects provides insight into a wide variety of astrophysics ranging from stellar interiors to galactic nuclei. The Zwicky Transient Facility (ZTF) provides time series observations that record the variability of more than a billion sources. The scale of these data necessitates automated approaches to make a thorough analysis. Building on previous work, this paper reports the results of the ZTF Source Classification Project (SCoPe), which trains neural network and XGBoost machine learning (ML) algorithms to perform dichotomous classification of variable ZTF sources using a manually constructed training set containing 170,632 light curves. We find that several classifiers achieve high precision and recall scores, suggesting the reliability of their predictions for 112,476,749 light curves across 40 ZTF fields. We also identify the most important features for XGB classification and compare the performance of the two ML algorithms, finding a pattern of higher precision among XGB classifiers. The resulting classification catalog is available to the public, and the software developed for SCoPe is open-source and adaptable to future time-domain surveys.
#### Spectroastrometry and Imaging Science with Photonic Lanterns on  Extremely Large Telescopes
 - **Authors:** Yoo Jung Kim, Michael P. Fitzgerald, Jonathan Lin, Steph Sallum, Yinzi Xin, Nemanja Jovanovic, Sergio Leon-Saval, Christopher Betters, Pradip Gatkine, Olivier Guyon, Julien Lozi, Dimitri Mawet, Barnaby Norris, Sébastien Vievard
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2312.00221
 - **Pdf link:** https://arxiv.org/pdf/2312.00221
 - **Abstract**
 Photonic lanterns (PLs) are tapered waveguides that gradually transition from a multi-mode fiber geometry to a bundle of single-mode fibers. In astronomical applications, PLs can efficiently couple multi-mode telescope light into a multi-mode fiber entrance and convert it into multiple single-mode beams. The output beams are highly stable and suitable for feeding into high-resolution spectrographs or photonic chip beam combiners. For instance, by using relative intensities in the output cores as a function of wavelength, PLs can enable spectroastrometry. In addition, by interfering beams in the output cores with a beam combiner in the backend, PLs can be used for high-throughput interferometric imaging. When used on an Extremely Large Telescope (ELT), with its increased sensitivity and angular resolution, the imaging and spectroastrometric capabilities of PLs will be extended to higher contrast and smaller angular scales. We study the potential spectroastrometry and imaging science cases of PLs on ELTs, including study of exomoons, broad-line regions of quasars, and inner circumstellar disks.
#### The IACOB project X. Large-scale quantitative spectroscopic analysis of  Galactic blue supergiants
 - **Authors:** Abel de Burgos, Sergio Simón-Díaz, Miguel A. Urbaneja, Joachim Puls
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2312.00241
 - **Pdf link:** https://arxiv.org/pdf/2312.00241
 - **Abstract**
 Blue supergiants (BSGs) are key objects for understanding the evolution of massive stars. However, discrepancies between theoretical predictions and empirical observations have opened up important questions yet to be answered. Studying statistically significant and unbiased samples of BSGs can help to improve the situation. We aim to perform a homogeneous and comprehensive quantitative spectroscopic analysis of a large sample of Galactic BSGs from the IACOB spectroscopic database. We derive the projected rotational velocity ($v\sin i$) and macroturbulent broadening ($v_{\rm mac}$) using IACOB-BROAD. We used FASTWIND computations to derive effective temperatures ($T_{\rm eff}$), surface gravities, microturbulences ($\xi$), Si and He surface abundances, and the wind-strength parameter. We provide estimates of the above-mentioned quantities for the largest sample of Galactic BSGs spectroscopically analyzed to date, comprising 538 O9-B5 type stars. We find a drop in the relative number of BSGs at ~21 kK, coinciding with a scarcity of fast rotating stars below that temperature. We speculate that this feature might be roughly delineating the location of the empirical Terminal-Age-Main-Sequence in the 15-85Msol range. By investigating the main characteristics of the $v\sin i$ distribution of O stars and BSGs as a function of $T_{\rm eff}$, we propose that an efficient mechanism transporting angular momentum from the stellar core to the surface might be operating along the main sequence. We find correlations between $\xi$, $v_{\rm mac}$ and the spectroscopic luminosity. We also find that no more than 20% of the BSGs have atmospheres enriched in He, and suggest that the origin of this specific sub-sample of BSGs might be in binary evolution. We do not find clear empirical evidence of an increase in the wind-strength over the wind bi-stability region towards lower $T_{\rm eff}$.

by olozhika (Xing Yuchen). 

2023-12-04
